### PR TITLE
Note that build_args has been renamed build-args

### PR DESCRIPTION
**Pull request by danlester**
Upgrade guide minor addition.

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 * Add [`outputs`](https://github.com/docker/buildx#-o---outputpath-typetypekeyvalue) input
 * Add [`cache-from`](https://github.com/docker/buildx#--cache-fromnametypetypekeyvalue) input (`cache_froms` removed)
 * Add [`cache-to`](https://github.com/docker/buildx#--cache-tonametypetypekeyvalue) input
+* Add `build-args` input (`build_args` removed)
```
Comment (crazy-max) [inline UPGRADE.md:16]:
```suggestion
* Rename `build_args` input to `build-args` for consistency with other Docker build tools
```

**Review by crazy-max — CHANGES_REQUESTED** [review]

**Review by crazy-max — CHANGES_REQUESTED** [review]
Thanks for your input. Small changes for better clarity and please sign your commit. `git commit --amend -s --no-edit`.

**Comment by danlester** [comment]
Thanks for your reply. I have attempted to make the change and also signoff the commit... not sure why the DCO test is still failing, sorry!

**Comment by crazy-max** [comment]
@danlester 

> Thanks for your reply. I have attempted to make the change and also signoff the commit... not sure why the DCO test is still failing, sorry!

That's because your previous commit is not signed. Squash your commits and create a new one signed.

**Comment by danlester** [comment]
I think I've done it! Thanks for your help, and at least I learnt something about git... Thanks for all your work on docker github actions.

**Review by crazy-max — APPROVED** [review]
@danlester LGTM thanks!

**Pull request merged**